app/components/article-featured-video.js

```diff
@@ -4,42 +4,48 @@ import Component from '@ember/component';
 import {on} from '@ember/object/evented';
 import {observer, computed} from '@ember/object';
 import {htmlSafe} from '@ember/string';
+import {throttle} from '@ember/runloop';
 import VideoLoader from '../modules/video-loader';
 import extend from '../utils/extend';
 import {transparentImageBase64} from '../utils/thumbnail';
 import config from '../config/environment';
-import {inGroup} from '../modules/abtest';
 import {track, trackActions} from '../utils/track';
+import duration from '../utils/duration';
 import JWPlayerMixin from '../mixins/jwplayer';
 
-// FIXME: After FeaturedVideo AB test is finished, consider removing inclusion of this mixin
-import RenderComponentMixin from '../mixins/render-component';
-
-const scrollClassName = 'is-on-scroll-video';
-
-export default Component.extend(RenderComponentMixin, JWPlayerMixin, {
+export default Component.extend(JWPlayerMixin, {
 	ads: service(),
 	wikiVariables: service(),
-	smartBanner: service(),
 
 	classNames: ['article-featured-video'],
+	classNameBindings: ['isOnScrollActive'],
 
-	// transparent gif
 	attributionAvatarUrl: transparentImageBase64,
-
-	smartBannerVisible: readOnly('smartBanner.smartBannerVisible'),
-	isFandomAppSmartBannerVisible: readOnly('smartBanner.isFandomAppSmartBannerVisible'),
+	isOnScrollActive: false,
 
 	initialVideoDetails: readOnly('model.embed.jsParams.playlist.0'),
 	currentVideoDetails: oneWay('initialVideoDetails'),
 	metadata: reads('model.metadata'),
 	placeholderImage: readOnly('initialVideoDetails.image'),
 	hasAttribution: and('currentVideoDetails.{username,userUrl,userAvatarUrl}'),
 
+	// initial video duration is in seconds, related video duration is a formatted string `MM:SS`
+	videoDuration: computed('currentVideoDetails', function () {
+		const currentVideoDuration = this.get('currentVideoDetails.duration');
+		if (this.get('currentVideoDetails') === this.get('initialVideoDetails')) {
+			return duration(currentVideoDuration);
+		}
+		return currentVideoDuration;
+	}),
+
 	placeholderStyle: computed('placeholderImage', function () {
 		return htmlSafe(`background-image: url(${this.get('placeholderImage')})`);
 	}),
 
+	isOnScrollActiveObserver: observer('isOnScrollActive', function () {
```

Comment:
this could be a typical function that you can call when needed
I think this would be much cleaner to read